presto-docs/src/main/sphinx/release/release-0.151.rst

```diff
@@ -0,0 +1,34 @@
+=============
+Release 0.151
+=============
+
+General Changes
+---------------
+
+* Fix bug where partial aggregation may be repeated when ``task.concurrency = 1``.
+* Fix query failure when ``array``, ``map``, or ``row`` type is used in non-equi ``JOIN``.
+* Fix performance regression for queries using ``OUTER JOIN``.
+* Fix query failure when using the :func:`arbitrary` aggregation function on ``integer`` type.
+* Add various math functions that operate directly on ``float`` type.
+* Add flag ``experimental.legacy-array-agg`` to restore legacy :func:`array_agg`
```

Comment:
This flag should not be called `experimental`. It's not experimental. Maybe name it `deprecated.legacy-array-agg`.

Also add "This flag will be removed in a future release."